AI-and-Analytics/Getting-Started-Samples/IntelPython_daal4py_GettingStarted/README.md

```diff
@@ -24,107 +24,130 @@ In this sample, you will run a batch Linear Regression model with oneDAL daal4py
 | Software                | Intel® oneAPI Data Analytics Library (oneDAL)
 > **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
 
-### For Local Development Environments
+## Key Implementation Details
 
-You will need to download and install the following toolkits, tools, and components to use the sample.
+- This get started sample code is implemented for CPUs using the Python language. The example assumes you have daal4py and scikit-learn installed inside a conda environment, similar to what is delivered with the installation of the Intel® Distribution for Python* as part of the Intel(R) AI Tools.
 
-- **Intel® AI Tools**
+- The Intel® oneAPI Data Analytics Library (oneDAL) is ready for use once you finish the Intel(R) AI Tools installation and have run the post installation script.
 
-  You can get the AI Tools from [Intel® oneAPI Toolkits](https://www.intel.com/content/www/us/en/developer/tools/oneapi/toolkits.html#analytics-kit). <br> See [*Get Started with the Intel® AI Tools for Linux**](https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux) for AI Kit installation information and post-installation steps and scripts.
+## Environment Setup
 
+You will need to download and install the following toolkits, tools, and components to use the sample.
 
-## Key Implementation Details
+**1. Get Intel® AI Tools**
 
-- This get started sample code is implemented for CPUs using the Python language. The example assumes you have daal4py and scikit-learn installed inside a conda environment, similar to what is delivered with the installation of the Intel® Distribution for Python* as part of the Intel® AI Analytics Toolkit.
+Required AI Tools: daal4py.

```

Comment:
There is no such tool on AI Tools Selector https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html. Maybe we need Scikit-learn here?

Reply:
Correct me if I'm wrong, but I'm assuming user can get daal4py by installing Intel® Extension for Scikit-learn* from AI Tools Selector. If yes, we need to state clearly that the required AI Tool is Intel® Extension for Scikit-learn*

Reply:
The Intel® Extension for Scikit-learn* will install daal4py because daal4py is a dependency, but to run this specific sample where daal4py is directly used, the developer does not need the Intel® Extension for Scikit-learn*. If the AI Tool selector does not have daal4py we could mention the Intel® Extension for Scikit-learn* as an alternative

Reply:
ok, then maybe we can say something like this:
```suggestion
Required AI Tools: daal4py (Select Intel® Extension for Scikit-learn* on [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to install)

```